packages/new-plugin/src/cli.js

```diff
@@ -317,6 +318,109 @@ async function processAnswers(answers) {
   series(processTemplate, renameExampleTemplate, renameDocsTemplate, renameReadmeTemplate)();
 }
 
-const cwdInfo = await getCwdInfo();
-const answers = await runPrompts(cwdInfo);
-await processAnswers(answers);
+async function runWithArgs(cwdInfo, options) {
+  // Check if package already exists
+  const packagePath = `${cwdInfo.destDir}/plugin-${getHyphenateName(options.name)}`;
+  if (fs.existsSync(packagePath)) {
+    console.error(`Error: A plugin package with this name already exists in this directory: ${packagePath}`);
+    process.exit(1);
+  }
+
+  // Set defaults and calculate derived values
+  const name = options.name;
+  const language = options.language || 'ts';
+  
+  let readmePath = options.readmePath;
+  if (!readmePath) {
+    if (!cwdInfo.isContribRepo) {
+      const remoteGitUrl = await getRemoteGitUrl();
+      readmePath = `${getGitHttpsUrl(remoteGitUrl)}/plugin-${getHyphenateName(name)}/README.md`;
+    } else {
+      readmePath = `https://github.com/jspsych/jspsych-contrib/packages/plugin-${getHyphenateName(name)}/README.md`;
+    }
+  }
+
+  return {
+    name: name,
+    description: options.description,
+    author: options.author,
+    authorUrl: options.authorUrl || '',
+    language: language,
+    readmePath: readmePath,
+    destDir: cwdInfo.destDir,
+    isContribRepo: cwdInfo.isContribRepo,
+  };
+}
+
+// Set up Commander.js
+const program = new Command();
+
+program
+  .name('new-plugin')
+  .description('Creates a new jsPsych plugin package')
+  .version('1.0.0')
```

Comment:
This version should match the version number in package.json. Can we import that to make it automatically update?